### ISSUE TITLE
Extremely long training time for Qwen2.5-14B SFT

### ISSUE THREAD

**Issue by 647sherry** [verl-project/verl#3612]
Epoch 1/2:   0%|                                                              | 0/3106 [00:00<?, ?it/s]
step:1 - train/loss:3.8543667793273926 - train/lr(1e-3):1.6103059581320453e-05 - train/time(s):958.4169
092178345
Epoch 1/2:   0%|                                                 | 1/3106 [16:03<831:18:12, 963.83s/it]

16min for 1 step, seems fine-tune 2 epochs need couple of months.
args:
set -x
export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5
torchrun --nproc_per_node=6 --nnodes=1 -m verl.trainer.fsdp_sft_trainer \
    data.train_files=train.parquet \
    data.val_files=test.parquet \
    data.max_length=4096 \
    data.train_batch_size=252 \
    data.prompt_key=extra_info \
    data.response_key=extra_info \
    data.prompt_dict_keys=['question'] \
    +data.response_dict_keys=['answer'] \
    data.micro_batch_size_per_gpu=3 \
    model.partial_pretrain=Qwen2___5-14B-Instruct \
    trainer.default_local_dir=ckpt_0912 \
    trainer.project_name=sft \
    trainer.experiment_name=sft-qwen-14b-instruct \
    trainer.total_epochs=2 \
    trainer.logger='["console"]' $@

fine-tune with 6*H20, how to improve? thx in advance

**Comment by Kirrito-k423** [comment]
中文：

该 issue 命中定时清理的默认关闭规则：issue 创建于 2025-09-25T06:31:56Z，早于本轮运行日期 2026-07-20 的 6 个月阈值（cutoff: 2026-01-20T00:00:00Z）；检查 comments 数量为 0，timeline 中未发现公开引用、PR/issue cross-reference、commit/PR mention 或其它公开关联互动。因此按定时清理标准关闭。

可追溯证据：
- Issue: https://github.com/verl-project/verl/issues/3612
- zero_comment_zero_reference_created_at: 2025-09-25T06:31:56Z
- comments_count: 0
- 已检查 comments 与 timeline 来源：https://api.github.com/repos/verl-project/verl/issues/3612/comments 和 https://api.github.com/repos/verl-project/verl/issues/3612/timeline

如果作者或社区成员仍能提供新的复现、证据或需求背景，可以重新评论请求重开，或创建新的 issue 并链接到这里。

English:

This issue matches the scheduled cleanup default-close rule: it was created at 2025-09-25T06:31:56Z, which is older than the six-month cutoff for this run (2026-01-20T00:00:00Z, run date: 2026-07-20); the checked comment count is 0, and the timeline did not show public references, PR/issue cross-references, commit/PR mentions, or other public linked activity. Therefore, this issue is being closed under the scheduled cleanup policy.

Traceable evidence:
- Issue: https://github.com/verl-project/verl/issues/3612
- zero_comment_zero_reference_created_at: 2025-09-25T06:31:56Z
- comments_count: 0
- Checked comments and timeline sources: https://api.github.com/repos/verl-project/verl/issues/3612/comments and https://api.github.com/repos/verl-project/verl/issues/3612/timeline

If the author or community can still provide a fresh reproduction, evidence, or additional context, please comment to request reopening or create a new linked issue.

This response was generated by a scheduled Codex GPT5.5 issue processing task.
